README.md

```diff
@@ -2377,25 +2377,53 @@ $user->getEmails();  // this will also return *only* active emails, as the user
 What are the options?
 
 1) change `->select('c, a')` to `->select('c')`, so we do not pre-fetch accounts;
-2) pre-fetch them with a separate join:
-
+2) pre-fetch them without filtering:  
+Keep in mind that pre-fetching is OK and recommended in most of the cases where we will use those relations.
+It's just not good if we also filter by that relation.  
+When pre-fetching relation there is no point for separate join as in the end results will contain all emails.  
```

Comment:
It makes sense, though, as not all users can have active emails (in this concrete example). This is also the most common use-case from my experience - I mean when filtering alters the result.

Reply:
Yep it does make sense :( Reverted back pre-filtering with separate join and added more details for that option.